hack/csv-generate.sh

```diff
@@ -43,8 +50,12 @@ fi
 # clean up all old data first
 clean_package
 
+# do not generate new CRD for old versions
+if [[ "$CSV_VERSION" != "$PREV" ]]; then
+  $OPERATOR_SDK generate crds
```

Comment:
What do you think about moving the CSV generation here as well?

Reply:
yep, moved